packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -377,9 +392,12 @@ class Auth {
         this.set('customer_type', isGuest ? 'guest' : 'registered')
 
         const refreshTokenKey = isGuest ? 'refresh_token_guest' : 'refresh_token_registered'
+        const expiresDate = res.refresh_token_expires_in
+            ? this.convertSecondsToDate(res.refresh_token_expires_in)
+            : undefined
```

Comment:
As you mentioned because our tests aren't all properly passing tokens it might be too much work at the moment to fix them, so we are just falling back to allowing the passing of undefined to the set function. We should look at fixing this in the future and we need to create a ticket for this to place in the backlog.